deploy/aws/deploy/dependencies/helmfile.yaml

```
@@ -0,0 +1,50 @@
+environments:
+  default:
+    values:
+      - namespace: "staging"
+      - staging.yaml
+  staging:
+    values:
+      - namespace: "staging"
+      - staging.yaml
+  prod:
```

you declared 'production' env in readme file
```suggestion
  production:
```